### PR TITLE
Add support for ember-cli-deploy

### PR DESCRIPTION
**Pull request by ryansch**

### CODE REVIEW

File: data/deploy.rb [hunk 1]
```diff
@@ -17,6 +18,9 @@
 set :deploy_to, lambda { "/home/#{settings.user!}/preflight/#{settings.app_name!}-#{settings.rails_env!}" }
 set :repository, ENV['REPOSITORY']
 set :branch, ENV['BRANCH']
+set :remove_git_dir, false
```
Comment (morganick) [inline data/deploy.rb]: What?

File: lib/ops_preflight/ember_tasks.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  task install: :environment do
+    in_directory './ember' do
+      queue! %[
+        ln -sf "#{deploy_to}/#{shared_path}/node_modules" "./node_modules"
+        ln -sf "#{deploy_to}/#{shared_path}/bower_components" "./bower_components"
+      ]
+      queue %[
+        echo "-----> Installing ember dependencies"
+        #{echo_cmd %[npm install bower ember-cli]} &&
+        #{echo_cmd %[npm install]} &&
```
Comment (morganick) [inline lib/ops_preflight/ember_tasks.rb]: What does this line do?

**Comment by morganick** [comment]
Comments left

File: data/deploy.rb (hunk 1 above)
Comment (ryansch) [inline data/deploy.rb]: This adds a configuration flag to control: https://github.com/mina-deploy/mina/pull/348

File: lib/ops_preflight/ember_tasks.rb (hunk 2 above)
Comment (ryansch) [inline lib/ops_preflight/ember_tasks.rb]: Runs `npm install`.  Please clarify your question.

Comment (morganick) [inline lib/ops_preflight/ember_tasks.rb]: Yeah, I know that much. I guess you are installing bower and ember-cli by name first and then installing the deps for the app. :+1:

Comment (ryansch) [inline lib/ops_preflight/ember_tasks.rb]: Yep.  As per our internal ember setup documentation.

**Comment by morganick** [comment]
:rocket:

**Pull request merged**